Gems/EMotionFX/Code/EMotionFX/Source/JointSelectionBus.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/std/containers/unordered_set.h>
+
+namespace EMotionFX
+{
+    class JointSelectionRequests
+        : public AZ::EBusTraits
+    {
+    public:
+        virtual const AZStd::unordered_set<size_t>& GetSelectedJointIndices() const = 0;
```

Comment:
We need to identify the actor instance here. Shall we add a parameter for that? So that we can request the selected joint indices per actor instance?

Reply:
this bus is basically trying to bridge the gap between emstudiomanager and runtime (because selection currently only happening in emstudio and debugDraw exist both runtime and editor).
The selectedJointIndices in emstudiomanager stores only the most active actorinstance's joint indices, because the we can only have one actor instance in selection.

Reply:
Perhaps a good way is to have the emstudio manager to remember the current selected instance, and do a check in the GetSelectedJointIndices funciton to check if the instance matches?